ivy_tests/test_ivy/test_functional/test_core/test_device.py

```diff
@@ -289,7 +289,7 @@ def test_function_supported_devices(
         res = ivy_backend.function_supported_devices(func)
         exp = set(expected)
 
-        assert sorted(tuple(exp)) == sorted(res)
```

Comment:
Are you sure this will pass the equality test?

Reply:
Yes, `exp` and `res` are already iterable objects, removing the `tuple()` should not change anything because the `sorted` function acts the same on any iterable. So, I think it will pass the equality test.